manifests/init.pp

```diff
@@ -193,6 +193,12 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing. This requires Foreman 1.22+
 #
+# $foreman_service_puma_threads_min::     Minimum number of threads for puma. Relevant only when Puma service is used and ignored when Passenger is used.
```

Comment:
Bit of a bikeshedding: should this leave out `puma` in the variable name? The use of `puma` vs `Puma` is also a bit inconsistent.

```suggestion
# $foreman_service_threads_min::     Minimum number of threads for Puma. Relevant only when Puma service is used and ignored when Passenger is used.
```

Reply:
I think calling out puma explicitly in the variable name will help the users under it as `puma threads` vs. just `foreman threads`.

Reply:
@ekohl Updated lowecase/uppercase inconsistencies, let me know what you think